src/main/java/gov/usgs/earthquake/distribution/EIDSNotificationSender.java

```diff
@@ -3,162 +3,40 @@
  */
 package gov.usgs.earthquake.distribution;
 
-import gov.usgs.earthquake.product.Product;
-import gov.usgs.earthquake.product.ProductId;
 import gov.usgs.earthquake.eidsutil.CorbaSender;
 import gov.usgs.util.Config;
 import gov.usgs.util.FileUtils;
 
 import java.io.File;
 import java.util.Date;
-import java.util.Iterator;
 import java.util.List;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-public class EIDSNotificationSender extends DefaultNotificationListener {
+public class EIDSNotificationSender extends DefaultNotificationSender {
 
 	/** Logging object. */
 	private static final Logger LOGGER = Logger
 			.getLogger(EIDSNotificationSender.class.getName());
 
-	/** Property referencing a product storage config section. */
-	public static final String PRODUCT_STORAGE_PROPERTY = "storage";
-
-	/** Property referencing how long to store products in milliseconds. */
-	public static final String PRODUCT_STORAGE_MAX_AGE_PROPERTY = "storageage";
-
-	/** Default max age to store products, 604800000 milliseconds = 1 week. */
-	public static final String DEFAULT_PRODUCT_STORAGE_MAX_AGE = "604800000";
-
-	/** Property referencing eids server host. */
-	public static final String EIDS_SERVER_HOST_PROPERTY = "serverHost";
-
-	/** Property referencing eids server port. */
-	public static final String EIDS_SERVER_PORT_PROPERTY = "serverPort";
-
 	/** Property referencing directory where notifications are "sent". */
 	public static final String EIDS_POLLDIR_PROPERTY = "serverPolldir";
 
 	/** Default directory where notifications are sent. */
 	public static final String EIDS_DEFAULT_POLLDIR = "polldir";
 
-	/** The product storage where retrieved products are stored. */
-	private URLProductStorage productStorage;
-
-	/** How long to store retrieved product, in milliseconds. */
-	private Long productStorageMaxAge;
-
 	/** How long to wait until checking for expired notifications/products. */
 	private Long senderCleanupInterval;
```

Comment:
I think this property is unused

Reply:
removed, along with getter/setter. No longer exists project-wide.